src/Terminus.php

```diff
@@ -138,23 +139,35 @@ public function run(InputInterface $input, OutputInterface $output)
         $status_code = $this->runner->run($input, $output, null, $this->commands);
         if (!empty($cassette) && !empty($mode)) {
             $this->stopVCR();
+        } else {
+            $this->runUpdateChecker();
```

Comment:
I still don't love that this outputs after the command instead of before. I understand the desire to have it not be ignored but this feels aggressive. All of the other cli's I have experience with put this warning first.